project/src/main/webapp/shared.js

```diff
@@ -146,29 +152,43 @@ function togglePrefilledInterests() {
 }
 
 /** Writes out relevant details to an event card */
-async function populateEventContainer(event, containerId) {
-  const indexOfEventCard = 25;
-  const eventCardTotal = await $.get('event-card.html');
+async function populateEventContainer(event, containerId, lod = 3) {
```

Comment:
Curious, what does lod stand for? Is it possible to give the longer name here or add javadoc to describe what it is?

Reply:
lod stands for level of detail - I think it's a general term/acronym, but I'll document it the first time it's mentioned